Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -2255,6 +2255,53 @@ BYE BYE BORDERS
       visibility: hidden;
     }
 
+    /*<ZEN-30423> white theme styles for reports start*/
+    .z-cse.z-cse-reports th.tabletitle div.tabletitlecontainer {
+        background-color: #d2d2d2 !important;
```

Comment:
I would prefer to have transparent background colors.  Then the background of the application behind the report will be shown.  We should discuss.

Reply:
you are right transparent background is that what we need there.